src/prices/PricesService.spec.ts

```diff
@@ -49,4 +61,23 @@ describe('PricesService', () => {
       eth: etherPrice,
     });
   });
+
+  it('Picks appropriate currency prices', () => {
```

Comment:
What do you think about refactoring this test to put the method in a describe block and nested behavior under it? 

For example:
```ts
describe('inCurrency', () => {
  describe('without a currency', () => {
    it('returns the usd price', () => {
      const defaultPrice = service.inCurrency(priceData);
      expect(defaultPrice).toBe(priceData.usd);
    });
  });
  // .
  // .
  // Add other flows here
  // .
  // .
});
```